tests/parsing/DefaultArgParserTest.py

```diff
@@ -31,3 +43,25 @@ def test_metavar_not_in_optional_args_sections(self):
                           flags=re.DOTALL)
         self.assertIsNotNone(match)
         self.assertEqual(match.group(1), '-a, --all')
+
+
+class AutocompleteTest(unittest.TestCase):
+
+    def setUp(self):
+        self._old_argcomplete = coalib.parsing.DefaultArgParser.argcomplete
+
+    def tearDown(self):
+        coalib.parsing.DefaultArgParser.argcomplete = self._old_argcomplete
```

Comment:
I don't really understand what the `setUp` and `tearDown` have achieved over here